state_presenter.go

```diff
@@ -27,21 +27,22 @@ func printOptions(state core.State, s tcell.Screen) {
 	var back views.Widget
 	var forth views.Widget
 
+	_, screenHeight := s.Size()
 	middle := views.NewCellView()
 
-	middle.SetModel(newVisualState(state))
+	middle.SetModel(newVisualState(state, screenHeight-1))
```

Comment:
Is this subtracting one for the status bar? I'd use a new const à la `const maxRows = screenHeight-1`.

Reply:
yes. It is for status bar. Has changed.